src/book.py

```diff
@@ -0,0 +1,96 @@
+import uuid
+import os
+import logging
+from datetime import datetime
+
+from flask import Blueprint, jsonify, request
+from cloudinary.uploader import upload
+from cloudinary.utils import cloudinary_url
+
+from src.models import Author, Book,  db
+from src.google import get_user_info
+from src.constants.http_status_codes import HTTP_201_CREATED, HTTP_400_BAD_REQUEST
+
+book_bp = Blueprint('book', __name__, url_prefix='/api')
+
+
+@book_bp.post('/books')
+def create_book():
+    file = request.files['image']
+    if not file:
+        return jsonify({
+            'error': "book image missing"
+        }), HTTP_400_BAD_REQUEST
+
+
+    # Author
+    book_author = request.form['author'].split(' ')
+    author_first_name = book_author[0]
+    author_last_name = ' '.join(book_author[1:])
+
+    # Book
+    title = request.form['title']
+    isbn = request.form['isbn']
+    language = request.form['language']
+    year_of_publication = request.form['year_of_publication']
+    category = request.form['category']
+    owner_id = request.form['owner_id']
+    
+    if not title:
+        return jsonify({
+            'error': "book's title missing"
```

Comment:
`book title is missing` or maybe `book title is required`
Similar comment for the rest of the fields.

Reply:
`"book title is required"`

Reply:
resolved